webpack.config.production.js

```diff
@@ -1,23 +1,23 @@
 /* eslint-disable */
 
-var path = require("path");
-var webpack = require("webpack");
+var path = require('path');
```

Comment:
oh okay but that should be in the separate PR, right? I'll just make sure that we use double quotes here.